node/test/Test/Pos/Client/Txp/UtilSpec.hs

```diff
@@ -257,6 +263,30 @@ feeForManyAddressesSpec inputSelectionPolicy manyAddrs =
         | manyAddrs = makeManyAddressesTo1Params
         | otherwise = makeManyUtxoTo1Params
 
+
+groupedPolicySpec :: HasTxpConfigurations => TxpTestProperty ()
+groupedPolicySpec =
+    testCreateMTx txParams >>= \case
+        Left err -> stopProperty $ pretty err
+        Right (txAux, _) ->
+            let picked = length . _txInputs . taTx $ txAux
+            in unless (picked == utxoNum) . stopProperty
+            $ sformat ("Only "%build%" inputs were used instead of all of the inputs") picked
+  where
+    utxoNum = 10 :: Int
+    txParams = makeManyUtxoTo1Params OptimizeForSecurity (fromIntegral utxoNum) 1000000 1
+
+ungroupedPolicySpec :: HasTxpConfigurations => TxpTestProperty ()
+ungroupedPolicySpec =
+    testCreateMTx txParams >>= \case
+        Left err -> stopProperty $ pretty err
+        Right (txAux, _) ->
+            let picked = length . _txInputs . taTx $ txAux
+            in unless (picked == (1 :: Int)) . stopProperty
```

Comment:
Is `:: Int ` needed here? `picked` seem to already be `Int`

Reply:
Nope, it is a leftover after me testing stuff.